scripts/unzip_wrapper.py

```diff
@@ -94,24 +129,19 @@ def main(search_path: str):
     # iterate over targets, set each one off in parallel
     for blobname, blobsize in blobs:
         # create and config job
-        job = get_batch().new_job(name=f'decompress {blobname}')
-        job.image(get_config()['workflow']['driver_image'])
-        job.cpu(4)
-        job.storage(f'{blobsize}Gi')
-        authenticate_cloud_credentials_in_job(job)
-        copy_common_env(job)
-        prepare_git_job(
-            job,
-            organisation='populationgenomics',
-            repo_name='analysis-runner',
-            commit=COMMIT_HASH,
-        )
+        job = set_up_batch_job(blobname, blobsize)
+
+        # read this blob into the batch
+        batch_blob = get_batch().read_input(path=blobname)
```

Comment:
From the Hailbatch docs: 
>read_input(self, path: str) -> _resource.InputResourceFile:

        Create a new input resource file object representing a single file.

        .. warning::

            To avoid expensive egress charges, input files should be located in buckets
            that are multi-regional in the United States because Batch runs jobs in any
            US region.

I think we can safely ignore this warning because our Hail batch cluster is located in Aus, but just thought I'd flag it.

Reply:
Hmm, I think this is a fault in the documentation... The warning is valid for most circumstances, but we host our own local instance so this is not relevant to us. AFAIK we may be the only Hail Batch deployment outside the core team in the US...

Reply:
MSFT also have a deployment, but we were the only for a while